source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+   - It will also be available within the UI, but it is always good to keep a copy.
+   - **Result**: Once you copied the key, the status will change to **Online**.
+8. Now that you have set up the ESPHome add-on, continue with one of the 2 different methods to add custom images:
+   - [Using images from a community repository](#using-images-from-a-community-repository)
+   - [Using your own images](#using-your-own-images)
+
+## Using images from a community repository
+
+- Perform this procedure, if you want to use ready-made images by the community.
+- If you want to use your own images, skip this procedure and go to [Using your own images](#using-your-own-images) instead.
+
+**Duration**: about 30 minutes
+
+### To use images from the community
+
+1. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+2. For inspiration, we have prepared some images for you.
+   - Check them out in this [repository](https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/tree/main).
+3. For this tutorial, we will use some images of Frenck.
+   - Add the following lines into the `substitution` block.
+
+   {% raw %}
+
+     ```yaml
+      loading_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/loading_320_240.png
+      idle_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/idle_320_240.png
+      listening_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/listening_320_240.png
+      thinking_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/thinking_320_240.png
+      replying_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/replying_320_240.png
+      error_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/error_320_240.png
+    ```
+
+   {% endraw %}
+   
+4. Save the changes and select **Install**:
+   - Depending on your environment, the installation process can take a while.
+   - Expect this to take about 25 minutes. The code needs to be compiled before it can be installed.
+![ESP32-S3-BOX config file](/images/assist/esp32-s3-config-05.png)
+   
+5. Once the installation is complete, you can see the new image on the S3-BOX.
+   - Reboot the device.
+   - Now, speak a command to test the new setting. For example, *OK Nabu turn on the light*.
+
+## Using your own images
+
+Perform this task if you want to use your own images on the S32-S3-BOX.
```

Comment:
Can we be a bit more enthusiasm and energy in this sentence ? Also, it's ESP32-S3-BOX (missing ES)